### PR TITLE
packaging: workaround missing crun dependency on criu-libs for libcriu.so.2

### PR DESCRIPTION
**Pull request by KKoukiou**
https://github.com/containers/crun/issues/1091

### CODE REVIEW

**Comment by KKoukiou** [comment]
Oh this needs update of the rhel-8-* images to include this package.

File: packaging/cockpit-podman.spec.in [hunk 1]
```diff
@@ -33,6 +33,9 @@ BuildRequires: libappstream-glib-devel
 
 Requires:       cockpit-bridge
 Requires:       podman >= 2.0.4
+%if 0%{?fedora} == 36 || 0%{?fedora} == 37 || 0%{?centos} == 9
```
Comment (martinpitt) [inline packaging/cockpit-podman.spec.in:36]: Can you please add a `# HACK:` link to the criu bug report here?

**Review by martinpitt — COMMENTED** [review]
Thanks!

File: packaging/cockpit-podman.spec.in (hunk 1 above)
Comment (KKoukiou) [reply packaging/cockpit-podman.spec.in:37]: Done :)

**Review by martinpitt — APPROVED** [review]
Epharisto!

**Pull request merged**